digdag-client/src/main/java/io/digdag/client/DigdagClient.java

```diff
@@ -671,6 +686,16 @@ public InputStream getLogFile(Id attemptId, String fileName)
                 .readEntity(InputStream.class);
     }
 
+    private WebTarget addDisableDirectDownloadParam(WebTarget target, boolean disableDirectDownload)
```

Comment:
Hmm, I don't want to use static method because we can't extend by override it.
And I also hope no side effective for ease of testing.
Anyway I will chose 
`private WebTarget addDisableDirectDownloadParam(WebTarget target)`

> Otherwise disableDirectDownload is misleading (is it private member field or argument?)
Sorry I can't understand your point out.

Reply:
@yoyama Context class private. No one can inherit it. It's not designed to be inherited.